plugins/hls-refactor-plugin/src/Development/IDE/GHC/ExactPrint.hs

```diff
@@ -542,6 +561,7 @@ annotate dflags needs_space ast = do
 
 -- | Given an 'LHsDecl', compute its exactprint annotations.
 annotateDecl :: DynFlags -> LHsDecl GhcPs -> TransformT (Either String) (LHsDecl GhcPs)
+#if !MIN_VERSION_ghc(9,2,0)
 -- The 'parseDecl' function fails to parse 'FunBind' 'ValD's which contain
```

Comment:
I'm leaning quite heavily on the tests here, but seems like this splitting then merging causes problems for on `>= 9.2`